File: 1560-number-of-students-doing-homework-at-a-given-time/1560-number-of-students-doing-homework-at-a-given-time.py
class Solution:
    def busyStudent(self, startTime: List[int], endTime: List[int], queryTime: int) -> int:
        result = 0
        for i in range(len(startTime)):
            if startTime[i]<=queryTime<=endTime[i]:
                result+=1
        return result

File: 1560-number-of-students-doing-homework-at-a-given-time/1560-number-of-students-doing-homework-at-a-given-time.java
class Solution {
    public int busyStudent(int[] startTime, int[] endTime, int queryTime) {
        int res = 0;
        for (int i = 0; i < endTime.length; i++) {
            int start = startTime[i];
            int end = endTime[i];
            if (start <= queryTime && queryTime <= end) {
                res++;
            }
        }
        return res;
    }
}